### ISSUE TITLE
type error with cheerio.load

### ISSUE THREAD

**Issue by brycedorn** [Bestulo/deno-cheerio#8]
Hi, I'm getting this type error when I use `cheerio.load()`:

```js
import { cheerio } from "https://deno.land/x/cheerio@1.0.6/mod.ts";

// Property 'load' does not exist on type '(selector: any, context: any, root: any, opts: any) => any'.
const $ = cheerio.load(html);
```

Functionally everything works fine. And this follows the example directly, am I missing something? Thanks.

**Comment by artze** [comment]
Facing the same issue as well.

**Comment by artze** [comment]
@Bestulo 
`cheerio` npm seems to already include TS typings. Tried the following and it seems to work:

```ts
// mod.ts

import * as cheerio from "https://jspm.dev/npm:cheerio";
export { cheerio };
```

With `mod.ts` above, the typings appear to be working but the `load` function now expects 2 args. By passing in `null` as the 2nd arg, e.g. `cheerio.load(html, null)`, I am able to get it to work, and tests passing. 

+++

I tested latest `cheerio` npm (v1.0.0-rc.12) on Node and the `load` function requires only 1 arg as expected. Not sure if `cheerio` hosted on `jspm.dev` reflects the latest version.

**Comment by sturmenta** [comment]
I ended up using:

`import * as cheerio from 'https://esm.sh/cheerio';`

**Comment by sturmenta** [comment]
I ended up using:

`import * as cheerio from 'https://esm.sh/cheerio@1.0.0-rc.12';`

<img width="1164" alt="Screen Shot 2022-09-30 at 20 32 55" src="https://user-images.githubusercontent.com/30802967/193369276-2edb119e-d406-4622-b944-44b4a1a4c2a4.png">

**Comment by Bestulo** [comment]
Is this a problem that you're having? I have some time now so I could just
push a fix if it would help. I have a pull request but I'm not sure if it's
the current fix.

**Comment by Bestulo** [comment]
Would #9 solve this problem?

**Comment by brycedorn** [comment]
> Would #9 solve this problem?

Yes, `1.0.0-rc.12` includes `types.d.ts` for proper typing: https://unpkg.com/browse/cheerio@1.0.0-rc.12/lib/types.d.ts

**Comment by Bestulo** [comment]
Merged #9 and solved this problem. This module is now deprecated (as long as cheerio remains fetcheable in working state from ESM). Please refer to the [README](https://github.com/Bestulo/deno-cheerio/blob/main/README.md) and messages posted in #9 for more information.
